lib/commands/element.js

```diff
@@ -40,6 +40,26 @@ commands.getText = async function (elementId) {
   return await this.uiautomator2.jwproxy.command(`/element/${elementId}/text`, 'GET', {});
 };
 
+commands.click = async function (elementId) {
+  return await this.uiautomator2.jwproxy.command(`/element/${elementId}/click`, 'POST', {elementId});
+};
+
+commands.tap = async function (elementId, x = 0, y = 0, count = 1) {
+  for (let i = 0; i < count; i++) {
+    if (elementId) {
+      // we are either tapping on the default location of the element
+      // or an offset from the top left corner
+      if (x !== 0 || y !== 0) {
+        await this.uiautomator2.jwproxy.command(`/element/${elementId}/click`, 'POST', {elementId, x, y});
+      } else {
+        await this.uiautomator2.jwproxy.command(`/element/${elementId}/click`, 'POST', {elementId});
+      }
+    } else {
+      await this.uiautomator2.jwproxy.command(`/element/${elementId}/click`, 'POST', {x, y});
```

Comment:
what do you think about adding custom one, something like `/session/:sessionId/appium/tap` and will take `x,y` coordinates as POST body i.e. `{ "x" : double, "y": double }`